### DISCUSSION TITLE
[feature] triggers for pulling in local-file

### DISCUSSION THREAD

**Discussion by abitawoods1** [cadeyrn/newtaboverride#324] [Ideas]
For those that modify the local page, it would be helpful it this add-on would auto-load on certain conditions:

a) on start-up of Firefox
b) on clicking your own toolbar extension icon

while "a" is helpful, b) would be most helpful: creating an icon that responds to a click by loading the stored local file URL

I modify my local home page, and tweek it to correct errors. The testing and iterations would be simple with the icon hit.

**Comment by cadeyrn** [comment]
WebExtensions can't access local file without explicit user interaction via an file input field for security reasons.
